### PR TITLE
readme modified

### PR DESCRIPTION
**Pull request by hassan-arif**
aur suna

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# test
\ No newline at end of file
+# test
+
+- hassan mahmood made this change
```
Comment (AnasMohammad4321) [inline README.md:3]: aur suna

Comment (hassan-arif) [reply README.md:3]: what

File: README.md [hunk 2]
```diff
@@ -1 +1,3 @@
-# test
```
Comment (AnasMohammad4321) [inline README.md:1]: why

**Review by hassan-arif — COMMENTED** [review]
what review

**Review by AnasMohammad4321 — COMMENTED** [review]
weird

**Comment by AnasMohammad4321** [comment]
Sae

**Pull request merged**